### PR TITLE
Channel messages view

### LINKED ISSUE #41: Create channel messages view
## User story

**Who:** **As a** user

**What:** **I want to** see the messages i have in a channel

**Why:** **so that we can** chat with other users

## Implementation details
- You are responsible for creating the functionality for making mockup 3 work
- Not creating the components, but putting the functionality together so it works
Here are some responsibilities:
- Fetching new messages every 5 seconds
- Rendering those messages using components others built
- Use context for the data handling
- You should glue the navigation in the top with the list of messages together with the input
- Talk with Tiago about his pr that sets up the routing

### PR DESCRIPTION
**Pull request by yukafjaponesa**
# Description

Please provide a short summary explaining what this PR is about.

Fixes #41 

# How to test?
use http://localhost:3000/channel/2 to see the messages fetched.

# Checklist

- [ x] I have performed a self-review of my own code
- [ x] I have commented my code, particularly in hard-to-understand areas
- [ x] I have made corresponding changes to the documentation
- [ ] This PR is ready to be merged and not breaking any other functionality

### CODE REVIEW

**Review by benna100 — COMMENTED** [review]
What is this story for?

File: src/client/containers/Channel/Channel.js [hunk 1]
```diff
@@ -1,35 +1,31 @@
...
-  }
-  //  return console.log(messages);
+  }, [channelId]);
+
+  return (
+    <ul className="message-list">
+      {messages.map(({ id, message }) => (
+        <li key={id}>{message}</li>
+      ))}
+    </ul>
```
Comment (benna100) [inline src/client/containers/Channel/Channel.js:27]: Please use the `ChannelMessagesView` here remember to import it
```suggestion
    <ChannelMessagesView messages={messages} />
```

**Review by benna100 — CHANGES_REQUESTED** [review]
Looks pretty good, small change and then its ready :)

File: src/client/containers/ChannelMessagesView/ChannelMessagesView.js [hunk 2]
```diff
@@ -0,0 +1,17 @@
+import React from 'react';
+import MessageList from '../../components/MessageList/MessageList';
+import PropTypes from 'prop-types';
+
+function ChannelMessagesView(props) {
+  return (
+    <ul className="container">
```
Comment (tpetrina) [inline src/client/containers/ChannelMessagesView/ChannelMessagesView.js:7]: If you are not styling it, then just rename the class name to `channel-messages`. The class name `container` is a bit too generic.

**Review by tpetrina — COMMENTED** [review]
Please rename class name and address Benjamin's comment.

**Pull request closed without merging**